doc/users/next_whats_new/type1decrypt.rst

```diff
@@ -0,0 +1,8 @@
+Type1Font objects now decrypt the encrypted part
```

Comment:
I wonder if this should be an API note rather than whats-new.  I'm not sure this is really a user-facing feature, even if it is public.